control/obstacle_collision_checker/include/obstacle_collision_checker/obstacle_collision_checker_node.hpp

```diff
@@ -87,11 +87,6 @@ class ObstacleCollisionCheckerNode : public rclcpp::Node
   NodeParam node_param_;
   Param param_;
 
-  // Dynamic Reconfigure
-  OnSetParametersCallbackHandle::SharedPtr set_param_res_;
-  rcl_interfaces::msg::SetParametersResult paramCallback(
-    const std::vector<rclcpp::Parameter> & parameters);
```

Comment:
This should not be removed and I do not understand why this fixes the issue.
Is another node setting the parameter `search_radius` to `0` ?

Reply:
I have the same question. I'd like to fix it properly.

Reply:
I updated `update_param` function, then parameters were set successfully. Also, I added node to `control.launch.py`. I am gonna change the PR name.